bin/ssh_scan_api

```diff
@@ -22,6 +22,18 @@ opt_parser = OptionParser.new do |opts|
     exit
   end
 
+  opts.on("--https", "Use HTTPS") do
+    options[:https] = true
+  end
```

Comment:
https would work like this
./bin/ssh_scan_api --https --https-key <keyfile> --https-crt <crtfile>

We were initially going to go for generation of self signed certs using OpenSSL everytime, but we thought you might want to add in one of LetsEncrypt based certificates on your own. If you want certs to be generated everytime we can add that back.